src/pages/EditorPage.tsx

```diff
@@ -150,23 +150,104 @@ export default function EditorPage(props: EditorPageProps): JSX.Element {
         },
         body: JSON.stringify(data),
       }
-    )
+    );
+  };
+
+  const [inputTab, setInputTab] = useAtom(inputTabAtom);
+  const tabsList = useAtomValue(tabsListAtom);
+  const inputTabIndex = useAtomValue(inputTabIndexAtom);
+  useEffect(() => {
+    if (inputTabIndex === tabsList.length) {
```

Comment:
should this be `>=` instead of `===`?

Reply:
I don't think that ever happens though? I assumed that when `tabsList` is recomputed, `inputTabIndex` is also recomputed, and it's always in the range `[0, tabsList.length]`.

Reply:
oh, makes sense 👍